src/models/converters/tests/transfer_erc20.rs

```diff
@@ -60,23 +58,23 @@ async fn erc20_transfer_dto_to_incoming_transfer_transaction_with_address_info()
     mock_info_provider.expect_safe_info().times(0);
     mock_info_provider.expect_token_info().times(0);
     mock_info_provider
-        .expect_full_address_info_search()
+        .expect_address_ex_from_any_source()
         .times(1)
-        .return_once(move |_| {
-            Ok(AddressInfo {
-                name: "".to_string(),
-                logo_uri: None,
+        .return_once(move |address| {
+            Ok(AddressEx {
+                value: address.to_string(),
+                name: Some("".to_string()),
```

Comment:
Judging by this change it seems like `name` used to not be nullable. Is that really the case? It doesn't matter too much but maybe we are being unnecessarily defensive here.

Reply:
This question was answered below :point_down: it was how it was originally served to the clients. And it makes sense as the `name` and `logo_uri` are the fallible parts of this `struct` if not resolved by the `info_provider`